results-tabulation-ui/src/components/election/extended-election/ProvincialCouncilElection2021/tally-sheet-edit/tally-sheet-edit-pe-21.js

```diff
@@ -0,0 +1,351 @@
+import React, {useState} from "react";
+import Table from "@material-ui/core/Table";
+import TableHead from "@material-ui/core/TableHead";
+import TableFooter from "@material-ui/core/TableFooter";
+import TableRow from "@material-ui/core/TableRow";
```

Comment:
These files can be excluded since not relevant

Reply:
ui side work is not finished yet